jcl/src/openj9.dtfj/share/classes/com/ibm/java/diagnostics/utils/plugins/impl/PluginManagerImpl.java

```diff
@@ -146,18 +148,16 @@ public void scanForClassFiles() throws CommandException {
 				}
 			}
 			
-			try {
-				logger.fine("Scanning path jrt:/modules/com.ibm.dtfjview in search of plugins");
-				FileSystem fs = FileSystems.getFileSystem(URI.create("jrt:/"));
-				Path modules = fs.getPath("modules/com.ibm.dtfjview");
-				listClassFiles(modules);
-			} catch (Exception e) {
-				//log and ignore exception or Java 8
-				logger.log(Level.FINE, "Error occurred scanning jrt:/modules/com.ibm.dtfjview.", e);
-			}
+			/*[IF Sidecar19-SE]*/
+			logger.fine("Scanning path jrt:/modules/openj9.dtfjview in search of plugins");
+			FileSystem fs = FileSystems.getFileSystem(URI.create("jrt:/"));
+			Path modules = fs.getPath("modules/openj9.dtfjview");
+			listClassFiles(modules);
+			/*[ENDIF]*/
```

Comment:
I think closing preprocessor tags should repeat the condition like we do for C/C++: i.e. this line should be:
```
/*[ENDIF] Sidecar19-SE */
```

Reply:
@keithc-ca That's an interesting thought but please have a wider discussion it before enforcing it in code reviews.  The project should strive for a consistent approach rather than a reviewer specific one =)

Reply:
Updated

Reply:
BTW, I note this practice is already being done elsewhere, as can be seen in #776